README.md

```diff
@@ -193,6 +193,28 @@ python llm_correctness.py \
 
 ```
 
+
+### AzureAI Compatible APIs
+```bash
+export AZUREAI_API_KEY=secret_abcdefg
+export AZUREAI_API_BASE="https://api.endpoints.anyscale.com/v1"
```

Comment:
Should this be an example Azure AI endpoint instead of an anyscale endpoint?

Reply:
updated the API_BASE to .ai.azure.com